.eslintrc

```diff
@@ -102,6 +102,7 @@
     "react/jsx-boolean-value": 0,
     "react/jsx-closing-bracket-location": 1,
     "react/jsx-curly-spacing": 1,
+    "react/jsx-indent": [1, 2],
```

Comment:
Yeah, they are definitely conflicting. Fix errors from one causes error from the other.